agent_logic.py

```diff
@@ -39,16 +44,28 @@ def map_response(text):
     return mapping.get(text.lower().strip(), None)
 
 def ask_questions(state):
+    import os
+    test_mode = os.getenv("TEST_MODE", "0") == "1"
+
     responses = {"PHQ9": [], "GAD7": []}
 
+    if test_mode:
```

Comment:
`test_mode` here is not explaining the intent. Is it something to be used only in `dev` phase? Or is it like a mental health test thing or something else.

Reply:
Right, thanks.